src/components/PostList.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { Post } from './Post';
+import { TypeOfPostList } from '../types';
+
+export const PostList = ({ list }) => (
+  <div style={{
+    background: 'dodgerblue', padding: '20px',
+  }}
```

Comment:
don't use inline styles. Style with classes and Classnames library

Reply:
changed